src/cascadia/WindowsTerminal_UIATests/Elements/TerminalApp.cs

```diff
@@ -166,11 +166,11 @@ private void ExitProcess()
 
             this.UIRoot = null;
 
-            if (this.job != IntPtr.Zero)
+            if (this.job == IntPtr.Zero)
             {
-                WinBase.TerminateJobObject(this.job, 0);
+                return;
             }
-            this.job = IntPtr.Zero;
+            WinBase.TerminateJobObject(this.job, 0);

```

Comment:
This isn't equivalent due to the lack of `this.job = ...`. I'd suggest:
```suggestion
            if (this.job != IntPtr.Zero)
            {
                WinBase.TerminateJobObject(this.job, 0);
                this.job = IntPtr.Zero;
            }
```